ws/templates/help/leaders/ws_gear.html

```diff
@@ -77,8 +77,13 @@ <h4>B Trip</h4>
   <ul>
     <li>All the group gear required for an A Trip +</li>
     <li>Two sleeping bags (one winter (15F) rated and one three-season (35F) rated minimum)</li>
-    <li>One bivy or bothy bag - A large MITOC tarp, an actual bothy bag, or a bivy bag such as the OR Alpine Bivy)</li>
-    <li>One shelter - A three-season or four-season tent and if weather conditions permit a bothy or tarp</li>
+    <li>One emergency shelter for 2 people, ex:</li>
+      <ul>
+        <li>Bothy bag (at least 2p)</li>
+        <li>Tent (at least 2p)</li>
+        <li>Bivy x2 (such as the OR Alpine Bivy, NOT an emergency blanket)</li>
```

Comment:
```suggestion
        <li>Bivy x2 (such as the Outdoor Research Alpine Bivy, <em>not</em> an emergency blanket)</li>
```

- Not everybody may know what OR stands for, it also seems like boolean "or"
- For accessibility/semantics, use `<em>` for emphasis